readme.md

```diff
@@ -371,6 +371,12 @@ try {
 }
 ```
 
+If you would like to use a modifier with a value not generated by Faker, use the `passthrough()` method. `passthrough()` simply returns whatever value it was given.
+
+```php
+$faker->optional()->passthrough('my custom value');
```

Comment:
please use a better example, e.g. by using `mt_rand` as the custom value generator.